linux/keyman-config/keyman_config/install_kmp.py

```diff
@@ -151,6 +151,8 @@ def install_kmp_shared(inputfile, online=False):
 				for kb in keyboards:
 					if kmx_id.lower() == kb['id'] and kmx_id != kb['id']:
 						os.rename(os.path.join(packageDir, f['name']), os.path.join(packageDir, kb['id']+'.kmx'))
+						fpath = os.path.join(packageDir, kb['id']+'.kmx')
+				extractico(fpath)
```

Comment:
Is this indented correctly (gee thanks Python)?

Reply:
I guess so. It's from @glasseyes 's original PR